### PR TITLE
Pnscan rule creation

### PR DESCRIPTION
**Pull request by signalblur**
### Summary of the Pull Request

I've created a new linux detector that detects the use of the FOSS Network Scanner [pnscan](https://github.com/ptrrkssn/pnscan). This activity has been observed by threat actors recently based on this intel: https://www.cadosecurity.com/blog/spinning-yarn-a-new-linux-malware-campaign-targets-docker-apache-hadoop-redis-and-confluence

Specifically it was observed that the TA executed: `/usr/local/bin/pnscan -t512 -R 6f 73 3a 4c 69 6e 75 78 -W 2a 31 0d 0a 24 34 0d 0a 69 6e 66 6f 0d 0a 221[.]0[.]0[.]0/16 6379`

This Sigma rule detects the the CLI pattern of `-R` or -`W` followed by hex characters (optionally preceeded by `"` as the tool documentation slightly differs from the intel, this pattern will match both)

Regex tested - https://regex101.com/r/RugQYK/1

### Changelog

new:	 Pnscan Binary Data Transmission Activity

### Example Log Event

N/A

### Fixed Issues
N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules/linux/process_creation/proc_creation_lnx_pnscan_binary_cli_pattern.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+references:
+    - https://www.cadosecurity.com/blog/spinning-yarn-a-new-linux-malware-campaign-targets-docker-apache-hadoop-redis-and-confluence
+    - https://regex101.com/r/RugQYK/1
+logsource:
+    category: process_creation
+    product: linux
+detection:
+    selection:
+        CommandLine:
+            - '.*(-(W|R)( |")(([0-9a-fA-F]{2}) ){2,20}).*'
```
Comment (frack113) [inline rules/linux/process_creation/proc_creation_lnx_pnscan_binary_cli_pattern.yml:16]: only the form, not the content
```suggestion
        CommandLine|re: '.*(-(W|R)( |")(([0-9a-fA-F]{2}) ){2,20}).*'
```

File: rules/linux/process_creation/proc_creation_lnx_pnscan_binary_cli_pattern.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+title: Detection of Pnscan Binary Data Transmission Behavior
+id: 97de11cd-4b67-4abf-9a8b-1020e670aa9e
+status: experimental
+description: Detects command line patterns associated with the use of pnscan for sending binary data across the network. This behavior has been identified in a Linux malware campaign targeting Docker, Apache Hadoop, Redis, and Confluence.
+author: David Burkett (@signalblur)
+date: 2024-04-03
```
Comment (frack113) [inline rules/linux/process_creation/proc_creation_lnx_pnscan_binary_cli_pattern.yml:6]: Hi,
https://github.com/SigmaHQ/sigma-specification/blob/main/Sigma_specification.md#date-optional
```suggestion
date: 2024/04/03
```

File: rules/linux/process_creation/proc_creation_lnx_pnscan_binary_cli_pattern.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+detection:
+    selection:
+        CommandLine|re: '.*(-(W|R)( |")(([0-9a-fA-F]{2}) ){2,20}).*'
+    condition: selection
+falsepositives:
+    - Legitimate use of pnscan for network diagnostics or authorized scanning activities.
+level: high
+tags:
+    - linux
+    - pnscan
```
Comment (frack113) [inline rules/linux/process_creation/proc_creation_lnx_pnscan_binary_cli_pattern.yml:22]: HI,
https://github.com/SigmaHQ/sigma-specification/blob/main/Tags_specification.md
```suggestion
tags:
    - attack.discovery
    - attack.t1046
```

**Review by frack113 — COMMENTED** [review]
fix workflow error

**Comment by nasbench** [comment]
Thanks for the submission just made some quick update to the regex to allow for single quotes and more spacing (as it's allowed). LGTM

**Review by nasbench — APPROVED** [review]

**Review by phantinuss — APPROVED** [review]

**Pull request merged**